envs/scIB-python.yml

```diff
@@ -48,11 +47,13 @@ dependencies:
   - r-testthat
   - r-vctrs
   - xlrd
+  - umap-learn==0.3.10
   - pip:
     - anndata==0.7.1
-    - trvae
-    - trvaep
-    - mnnpy
+    - trvae==1.1.2
+    - trvaep==0.1.0
+    - scanorama+git://github.com/danielStrobl/scanorama@patch-1
```

Comment:
I had to change this to `git+git://github.com/danielStrobl/scanorama@patch-1` to get it to work.